src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { fetchData } from '../Utils/api';
+
+const ArticleCard =() => {
+    const { article_id } = useParams()
+    const [article, setArticle] = useState([])
```

Comment:
It would be great to have a loading state that will display a loading message or icon if still loading the date :)